docs/rancher/node/node-driver.md

```diff
@@ -58,6 +58,12 @@ When the Harvester node driver is enabled, you can create Kubernetes clusters on
 
 :::
 
+## Known Issues
+
+| Summary                                                                                                                                             | Status                                                                                          | Last Updated |
+|-----------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|--------------|
+| [Rancher loses ability to manage/scale guest clusters when the corresponding API tokens expire](https://github.com/harvester/harvester/issues/5827) | [Temporary Workaround Exists](https://harvesterhci.io/kb/modify_harvester_cloud_credential_ttl) | 2024-05-17   |
```

Comment:
```suggestion
### Rancher loses the ability to manage or scale guest clusters when the corresponding API tokens expire.

**Issue**: [#5827](https://github.com/harvester/harvester/issues/5827)

**Description**: Rancher uses kubeconfig tokens to provision guest Kubernetes clusters on Harvester. When a kubeconfig token expires, you will no longer be able to perform management operations for the corresponding Rancher-managed guest Kubernetes cluster. This issue affects only Harvester cloud credentials created after installing or upgrading to Rancher 2.8.x, which introduced the [kubeconfig-default-token-ttl-minutes](https://ranchermanager.docs.rancher.com/api/api-tokens#kubeconfig-default-token-ttl-minutes) setting.

**Status**: A [temporary workaround](https://harvesterhci.io/kb/modify_harvester_cloud_credential_ttl) is available.

**Last updated**: 2024-05-17
```

Reply:
I suggest that we use this structure to somewhat align with other parts of the documentation (e.g., [Upgrade](https://docs.harvesterhci.io/v1.4/upgrade/v1-1-1-to-v1-1-3/#known-issues) and [Witness Node](https://docs.harvesterhci.io/v1.4/advanced/witness/#known-issues)) and to make room for extra details, including screenshots.

Reply:
I've updated the the structure for the v1.4/dev, v1.3/latest and v1.2 versions but left the v1.1 version in the table structure, since there was already another known-issue in the table at that exact location in the docs.